packages/stylelint-config-airbnb/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  plugins: [
+    'stylelint-scss',
+  ],
+  rules: {
+    'at-rule-empty-line-before': 'never',
```

Comment:
right but it's still appearing in an eslint config, so it could still conflict with a core *eslint* rule. i'm saying core styleint rules should be prefixed with "stylelint/"